chart/env/dev.yaml

```diff
@@ -86,6 +86,22 @@ api:
 
 # --- workers ---
 
+configNames:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 0.01
+    limits:
+      cpu: 1
+
+splitsNames:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 0.01
+    limits:
+      cpu: 1
```

Comment:
this is in case we want to deploy a dev cluster on ephemeral ?

Reply:
Yes, that's true